DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -50,18 +64,25 @@ public class Ball_Behavioiur : MonoBehaviour {
 		if (GameStart == true) {
 			Ball.transform.localScale = largeballscale;
 			GameStart = false;
-		} 
+		}
 
+        position = rb.transform.position;
 
-	}
+    }
 
     private void OnCollisionEnter2D(Collision2D coll)
     {
         if (coll.gameObject.tag == "Ball")
         {
             circle.isTrigger = true;
-
         }
+
+		if (coll.gameObject.tag == "Bottom") {
+			newSpeed.Set((float)rb.velocity.x,  Mathf.Sqrt ((float)(2 * 9.8 * (maxHeight - (-2.55)))));
```

Comment:
this line of code, finds the velocity it needs to make it so that the maxHeight is the apex. The -2.55 is the level of the ground, meaning that i would take the maxHeight and subtract it by the level of the ground to find the distance and plug into the formula